src/Weather/Rasp/RaspCache.cpp

```diff
@@ -39,9 +39,9 @@ RaspCache::RaspCache(const RaspStore &_store, unsigned _parameter) noexcept
 RaspCache::~RaspCache() noexcept = default;
 
 static constexpr unsigned
-ToHalfHours(BrokenTime t)
+ToQuarterHours(BrokenTime t)
 {
-  return t.hour * 2u + t.minute / 30;
+  return t.hour * 4u + t.minute / 15;
 }
```

Comment:
This is a rather intrusive and fragile change. This shouldn't be hidden inside a commit that pretends to just add a new RASP provider.